4_Zadania_dodatkowe/6.InfiniteSlider/css/style.css

```diff
@@ -1,10 +1,21 @@
-.slider {
-  height: 250px;
+.slider{
+  height: 200px; /* zmiana z 250 na 200 bo przy 250 widac czesc 2 obrazka */
+  width: 400px;
+  overflow: hidden;
 }
 
-.slider ul {
+.slider ul{
+  position: relative;
   list-style: none;
+  width: 3228px;
+  margin: 0;
+  padding: 0;
+  right: 404px;
```

Comment:
I think it should be handled in `js`

Reply:
updated - now both right and width is set by jquery